.github/workflows/compose-packer-verify.yaml

```diff
@@ -113,8 +113,9 @@ jobs:
         shell: bash
         # yamllint disable rule:line-length
         run: |
+          echo "${{ env.CLOUDS_ENV_B64 }}" | sed 's/^ */::add-mask::/'
```

Comment:
Could you explain how this is adding the mask? You pipe the echoed string into sed, but you don't seem to be capturing the sed output anywhere. Should this be assigned to the `clouds_env_b64` var?

Reply:
Based on our Slack conversation, I now understand that the intention is to print these lines to stdout such that the runner will pick them up as a workflow command. LGTM!